test/wrapper_array.jl

```diff
@@ -255,6 +255,17 @@ end
     @test stride(nda, :b) == 3 == stride(nda, 2) == stride(x, 2)
 end
 
+@testset "show" begin
+    nda = NamedDimsArray([1 2; 3 4], (:x, :y))
+    @test occursin("(:x, :y)", string(nda))
+    ndv = NamedDimsArray([1,2,3], :x)
+    @test occursin("], :x)", string(ndv))
+
+    str = repr(MIME"text/plain"(), nda)
+    @test occursin("NamedDimsArray(::", str)
+    @test occursin("→ :y", str)
```

Comment:
as well as having these `occursin` tests, which check certain key patterns are present, it would also be good to test some full (simple, obviously correct) cases
e.g. `@test repr(NamedDimsArray(1:3, :v)) == "..."`

Reply:
also, for `repr` we should test the output is parseable and evaluates back to the input (at least when `parent(nda) isa Array`s; other AbstractArrays may not have defined their own `repr` to be parseable back to its input)
i.e. `@test eval(Meta.parse(repr(nda))) == nda`